### DISCUSSION TITLE
Add minimize button to top menu when fullscreen

### DISCUSSION THREAD

**Discussion by Airmour** [rustdesk/rustdesk#5157] [Feature Request]
When in full screen mode, I want to hide the remote computer, I need to exit full screen first, and then click the minimize button to hide the program, it's not convenient.
![image](https://github.com/rustdesk/rustdesk/assets/20551803/2839131f-26e1-4f55-b905-e94c06b65585)

**Comment by rustdesk** [comment]
@fufesou make sense, please double check TeamViewer / AnyDesk also.

**Comment by fufesou** [comment]
@Airmour Thanks. https://github.com/rustdesk/rustdesk/pull/5166

**Comment by bauhuynhdev** [comment]
Looking forward to this feature, thanks to the development team.
